File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
    var answer = "";
    var cnt = new Array(26).fill(0);
    for(var i=0; i<s.length; i++){
        cnt[s.charCodeAt(i)-97]++;
    }
    for(var i=0;  i<26; i++){
        if(cnt[i]===1) answer += String.fromCharCode(i+97);
    }
    return answer;
}

File: 프로그래머스/lv0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.*;

import java.util.*;
class Solution {
    public String solution(String s) {
        String answer = "", ch = "";
        List<String> list = new ArrayList<>();
        for(int i=0; i<s.length(); i++){
            ch = Character.toString(s.charAt(i));
            System.out.println("ch "+ch);
            if(answer.contains(ch)) {
                list.add(ch);
            }
            answer += ch;
        }
        Iterator<String> iterator = list.iterator();
 
        while (iterator.hasNext()) {
            answer = answer.replace(iterator.next(),"");
        } 
        char[] arr = answer.toCharArray();
        Arrays.sort(arr);
        answer = String.valueOf(arr);
        return answer;
    }
}